gobblin-service/src/test/java/org/apache/gobblin/service/modules/orchestration/OrchestratorTest.java

```diff
@@ -152,6 +151,32 @@ private TopologySpec initTopologySpec() {
   }
 
   private FlowSpec initFlowSpec() {
```

Comment:
where is this function being used? are we asserting somewhere that this flow does compile?

Reply:
Yes, we add orchestrator as listener of specCatalog, which will compile the flow. So on line 273 -290, we test that if flow cannot compile, we will not be able to add that to the Flow catalog, and if flow can compile, we will put it in flowCatalog successfully.